init.lua

```diff
@@ -11,41 +9,40 @@ local income_msg = "You receive your citizen's income (+" .. income .. ")"
 
 local next_payout = os.time() + income_interval
 
-local ex = dofile(modpath .. "exchange.lua")
-local exchange = ex.open_exchange(savepath)
+local exchange = (dofile(modpath .. "exchange.lua")).open_exchange(
+	minetest.get_worldpath() .. "/global_exchange.db"
+)
```

Comment:
Is it a huge style issue that I didn't inline these? If I add new functions to the exchange.lua table then I will have to revert it (though I don't know if I will).